### PR TITLE
Removed unused CSS and added header component

### PR DESCRIPTION
**Pull request by TeresitaCGNader**
Removed CSS that we were not using from the default React.js template
Added Header component and styles
Now header shows on all pages, but I am working on aligning the logo or maybe I will end up using Photoshop to modify the logo so it fits our needs better.

### CODE REVIEW

**Comment by stevenbertolucci** [comment]
The changes looks good, but it won't merge because your branch has conflicts that must be resolved. I am not good on how to merge branches with conflicts. But, when I have branch conflicts,  I usually copy and paste the updated files from the main branch into my branch and merge from there. In your case,  I would copy the updated files from App.js and App.css from the main branch and paste into your branch then merge from there. This should resolve the conflicts in your branch.

**Review by TeresitaCGNader — COMMENTED** [review]
@cdgco I think part of your code is called Header which is interfiering with the header I commited, can you take a look?

**Comment by NickFranciscoOSU** [comment]
To fix the error I think you need to change the header import in app.js.
It seems to work but you may want to check?
The commented out one is what was there, and I replaced it with the line below.

//import { Header } from "react-bootstrap/lib/Modal.js";
import  Header from "./components/Header/Header.js";

image

File: src/App.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ import SignIn from "./pages/signin.js";
 import Dashboard from "./pages/shelterDashboard.js";
 import Pet from "./pages/pet.js";
 import FullScreenLoader from './components/FullScreenLoader';
+import { Header } from "react-bootstrap/lib/Modal.js";
```
Comment (cdgco) [inline src/App.js:19]: Looks like you are just importing the wrong `Header` component here.

Comment (TeresitaCGNader) [reply src/App.js:19]: That is not my code

I am using import Header from './components/Header/Header.js'

**Pull request closed without merging**